### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses false positive for union types (master branch)

### ISSUE THREAD

**Issue by LastDragon-ru** [slevomat/coding-standard#1162]
```php
<?php declare(strict_types = 1);

namespace Test;

use Illuminate\Database\Eloquent\Model;
use stdClass;

class Bug {
    protected function column(Model|stdClass|array|null $item): mixed {
        return $item;
    }
}
```

```shell
$ ./vendor/bin/phpcs ./packages/Bug.php

FILE: /project/packages/Bug.php
-----------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-----------------------------------------------------------------------------------
 5 | ERROR | [x] Type Illuminate\Database\Eloquent\Model is not used in this file.
 6 | ERROR | [x] Type stdClass is not used in this file.
-----------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------

Time: 595ms; Memory: 12MB
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/790822f62ab4209d88459be75de780f1734440b5
